### PR TITLE
Rotational inertia Tcl test port to Python

### PR DESCRIPTION
**Pull request by bogdan-tanygin**
Just a port of the `rotational_inertia.tcl`.
- no random parameters (stability)
- no notable performance impact: `Ran 1 test in 0.040s`

PR Checklist
------------
 - [ ] Tests?
   - [ ] Interface
   - [ ] Core 
 - [ ] Docs?

### CODE REVIEW

File: testsuite/python/rotational_inertia.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+if "MASS" in espressomd.features() and "ROTATIONAL_INERTIA" in espressomd.features():
+    class ThermoTest(ut.TestCase):
+        longMessage = True
+        # Handle for espresso system
+        es = espressomd.System()
+
+        def define_rotation_matrix(self, part):
+            A = np.zeros((3,3))
+            quat = self.es.part[part].quat
+            quatu = self.es.part[part].director
```
Comment (hmenke) [inline testsuite/python/rotational_inertia.py]: `quatu` seems to be unused.

File: testsuite/python/rotational_inertia.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+            L_body_return = self.es.part[part].omega_body[:] * self.es.part[part].rinertia[:]
+            return L_body_return
+            
+        def test(self):
+            self.es.cell_system.skin = 0
+            self.es.part.add(pos=np.array([0.0, 0.0, 0.0]),id=0)
+            
+            #Inertial motion around the stable and unstable axes
+            
+            self.es.thermostat.turn_off()
```
Comment (hmenke) [inline testsuite/python/rotational_inertia.py]: You do not need to turn off the thermostat if you never set one.

File: testsuite/python/rotational_inertia.py [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+            # ----------------------------
+            self.es.time_step = 0.0006
+            # Stable omega component should be larger than other components.
+            stable_omega = 57.65
+            self.es.part[0].omega_body=np.array([0.15, stable_omega, -0.043])
+            self.es.part[0].rinertia = J[:]
+            self.es.part[0].ext_torque = T[:]
+            
+            # Angular momentum
+            L_0_body = np.zeros((3))
```
Comment (hmenke) [inline testsuite/python/rotational_inertia.py]: Redundant.

File: testsuite/python/rotational_inertia.py [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+            vec_lab = np.zeros((3))
+            A = self.define_rotation_matrix(part)
+            vec_lab[:] = A[0,:] * vec[0] + A[1,:] * vec[1] + A[2,:] * vec[2]
+            return vec_lab
+            
+        # Angular momentum
+        def L_body(self, part):
+            L_body_return = np.zeros((3))
+            L_body_return = self.es.part[part].omega_body[:] * self.es.part[part].rinertia[:]
+            return L_body_return
```
Comment (hmenke) [inline testsuite/python/rotational_inertia.py]:
````python
def L_body(self, part):
    return self.es.part[part].omega_body[:] * self.es.part[part].rinertia[:]
````

File: testsuite/python/rotational_inertia.py [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+            A[2,0] = 2 * (quat[1] * quat[3] + quat[0] * quat[2])
+            A[2,1] = 2 * (quat[2] * quat[3] - quat[0] * quat[1])
+            
+            return A
+
+        def convert_vec_body_to_space(self, part, vec):
+            vec_lab = np.zeros((3))
+            A = self.define_rotation_matrix(part)
+            vec_lab[:] = A[0,:] * vec[0] + A[1,:] * vec[1] + A[2,:] * vec[2]
+            return vec_lab
```
Comment (hmenke) [inline testsuite/python/rotational_inertia.py]:
````python
def convert_vec_body_to_space(self, part, vec):
    A = self.define_rotation_matrix(part)
    return np.dot(A, vec)
````

**Review by hmenke — CHANGES_REQUESTED** [review]

File: testsuite/python/rotational_inertia.py (hunk 1 above)
Comment (bogdan-tanygin) [reply testsuite/python/rotational_inertia.py]: Agree

Comment (psci2195) [inline testsuite/python/rotational_inertia.py]: Agree, for a transposed `A`

File: testsuite/python/rotational_inertia.py (hunk 5 above)
Comment (bogdan-tanygin) [reply testsuite/python/rotational_inertia.py]: Agree, for a transposed `A`

File: testsuite/python/rotational_inertia.py (hunk 4 above)
Comment (bogdan-tanygin) [reply testsuite/python/rotational_inertia.py]: Agree

File: testsuite/python/rotational_inertia.py (hunk 2 above)
Comment (bogdan-tanygin) [reply testsuite/python/rotational_inertia.py]: Agree

File: testsuite/python/rotational_inertia.py (hunk 3 above)
Comment (bogdan-tanygin) [reply testsuite/python/rotational_inertia.py]: Agree

**Review by hmenke — APPROVED** [review]

**Comment by bogdan-tanygin** [comment]
Done..

**Pull request merged**